src/main/java/optic_fusion1/mcantimalware/check/checks/ExGrief.java

```diff
@@ -0,0 +1,20 @@
+package optic_fusion1.mcantimalware.check.checks;
```

Comment:
Can't verify this check due to lack of jar to test with, you'll have to provide or link to one

Reply:
Download cashploit!

Reply:
https://git.liquidbounce.net/CCBlueX/Exrief

Reply:
Thx

Reply:
Compiled ExGrief jar doesn't get detected

Reply:
Force-OP detects it however. Most likely due to the string you're checking for

Reply:
Idk it works for me.
[MALICIOUS] ExGrief.jar MIGHT be infected with ExGrief(Malware) Class Path: me/konsolas/aac/command/commands/CopyPluginCommand

Reply:
🤷‍♂

Reply:
You should still merge all the cleanups tho

Reply:
I'm aware, i'll just merge everything and then fix it *shrug*

Reply:
only because I'm fucking confused and can't be bothered to wait for a commit fixing it right now

Reply:
and no, doesn't mean i'll treat other PRs the same way, make sure your shit works..

Reply:
No need to swear... Just be happy that people are trying to help

Reply:
I am 👌 
Just saying.
The only reason i'm merging and dealing with this commit my self is because it _should_ work, but for some reason doesn't, still think it's just the string that's the issue though

Reply:
and if it's not the string that's the issue...then i'm just going to be even more confused honestly

Reply:
Maybe you didn't compile Exrief correctly?

Reply:
I did, though i have an idea on what the issue MIGHT be, since striping everything except Exrief from the string also didn't detect it

Reply:
Check#classNodeContainsBlacklistedWord switching equalsIgnoreCase to contains work, this will also cause false-positives in some cases.

Reply:
and admittedly, there's already enough false positives from #22 and people abusing setOp. There really doesn't need to be more

Reply:
Went through all current checks, false-positives shouldn't be an issue, however that could change in the future

Reply:
speaking of false-positives and since i have this webhooked to discord.
Note to self:
Find time to mass download a few thousand plugins and get around to seeing what else false-positives